hub/src/server/authentication.js

```diff
@@ -230,6 +240,23 @@ export class V1Authentication {
       throw new ValidationError('Must provide `iss` claim in JWT.')
     }
 
+    // check for revocations
+    if (options && options.oldestValidTokenTimestamp && options.oldestValidTokenTimestamp > 0) {
```

Comment:
We're probably going to have to schedule a "flag day" whereby all future tokens need to have issued-at timestamps.  Maybe that can be encoded here?  i.e. we allow you to use a token without a timestamp up until a given date, and then require it?  This way, we can roll out the Gaia hub to everyone before rolling out the required changes to blockstack.js.

Reply:
Or maybe I'm misunderstanding -- is `oldestValidTokenTimestamp` taken from the Gaia hub's config file?

Reply:
`options.oldestValidTokenTimestamp` is coming from the special auth file stored in the gaia bucket / LRU cache. 
So currently, if bucket owner has not issued any revocations, then the token issued-date will not be checked.

Reply:
Yup, that sounds even better :+1: